charts/quickwit/templates/job-create-sources.yaml

```diff
@@ -27,6 +27,10 @@ spec:
       securityContext:
         {{- toYaml $.Values.podSecurityContext | nindent 8 }}
       restartPolicy: Never
+      {{ - with .Values.initContainers }}
```

Comment:
To remain consistent with the other comments, this is to replace by:

```
{{- with .Values.jobs.sources.initContainers }}
```

And define a default value in the values.yaml file:

```yaml
jobs:
  sources:
    initContainers: []
```